src/main/java/Bebe.java

```diff
@@ -0,0 +1,149 @@
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Prints the available commands and their descriptions.
+     */
+    private static void showHelp() {
+        System.out.println("Here are the commands you can use:");
+        System.out.println("  todo <task description>          - Adds a ToDo task.");
+        System.out.println("  deadline <task description> /by <date/time>  - Adds a Deadline task.");
+        System.out.println("  event <task description> /from <start time> /to <end time>  - Adds an Event task.");
+        System.out.println("  list                            - Lists all tasks.");
+        System.out.println("  mark <task number>              - Marks a task as done.");
+        System.out.println("  unmark <task number>            - Marks a task as not done.");
+        System.out.println("  help                            - Shows this help message.");
+        System.out.println("  bye                             - Exits the chatbot.");
+    }
+
+    /**
+     * Runs the main loop of the chatbot to handle user input.
+     */
+    private static void runChatbot() {
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+
+        while (true) {
+            userInput = scanner.nextLine().trim();
+            String[] words = userInput.split(" ", 2);
+
+            switch (words[0].toLowerCase()) {
+                case "bye":
+                    scanner.close();
+                    return; // Exits the method, thus ending the program
+                case "list":
+                    listTasks();
+                    break;
+                case "mark":
+                    if (words.length == 2) {
+                        int taskNumber = Integer.parseInt(words[1]) - 1;
+                        markTaskAsDone(Integer.parseInt(words[1]));
+                    }
+                    break;
+                case "unmark":
+                    if (words.length == 2) {
+                        int taskNumber = Integer.parseInt(words[1]) - 1;
+                        markTaskAsNotDone(Integer.parseInt(words[1]));
+                    }
+                    break;
+                case "todo":
+                    addTask(new Todo(words[1]));
+                    break;
+                case "deadline":
+                    String[] deadlineParts = words[1].split(" /by ");
+                    addTask(new Deadline(deadlineParts[0], deadlineParts[1]));
+                    break;
+                case "event":
+                    String[] eventParts = words[1].split(" /from | /to ");
+                    addTask(new Event(eventParts[0], eventParts[1], eventParts[2]));
+                    break;
+                case "help":
+                    showHelp();
+                    break;
+                default:
+                    System.out.println("I'm sorry, I don't understand that command.");
+                    break;
+            }
```

Comment:
The case labels and their corresponding code blocks should be aligned with the switch statement itself. Consider setting your IDE to follow this indentation convention.